samples/istio/sidecar-mode/README.md

```diff
@@ -0,0 +1,107 @@
+# Istio - Sidecar Mode
+
+In this scenario, we deploy Choreo Connect within the Istio service mesh where it applies API Management for the microservices.
+
+### Installation Prerequisites
+
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+- [Kubernetes v1.20 or above](https://Kubernetes.io/docs/setup/) <br>
+
+    - Minimum CPU : 6vCPU
+    - Minimum Memory : 6GB
+
+- [Istio v1.11.x or above](https://istio.io/docs/setup/platform-setup/)
+
+### 1. Enable Istio Injection for the namespace
+
+```
+kubectl label namespace default istio-injection=enabled
```

Comment:
is the namespace `default` here?

Reply:
Yes, Use the default namespace which comes by default.